tests/test_misc.py

```diff
@@ -32,10 +32,13 @@ def send_email(msg):
     assert app.mail_sent is True
 
 
-def test_register_blueprint_flag(app, sqlalchemy_datastore):
```

Comment:
@Jaza can you keep this test case so we are sure `register_blueprint=False` works for existing projects?